runtime/karura/src/lib.rs

```diff
@@ -854,13 +854,15 @@ parameter_types! {
 	pub const GetStableCurrencyId: CurrencyId = KUSD;
 	pub const GetLiquidCurrencyId: CurrencyId = LKSM;
 	pub const GetStakingCurrencyId: CurrencyId = KSM;
+	pub Erc20HoldingAccount: H160 = H160(hex_literal::hex!("000000000000000000ff00000000000000000000"));
```

Comment:
This address looks pretty arbitrary to me. Can we use an account derived from currencies module pallet id?

Reply:
we want to use a system account, that's the one starts `000000000000000000`

I think we want to have this defined around here https://github.com/AcalaNetwork/Acala/blob/ded6de57234c4367401dbd758db609254c2e00e0/primitives/src/evm.rs#L129-L153